src/g_umapinfo.c

```diff
@@ -219,19 +211,6 @@ static char *ParseMultiString(scanner_t *s)
 {
     char *build = NULL;
 
-    if (SC_CheckToken(s, TK_Identifier))
```

Comment:
Why was this part removed?

Reply:
I replaced all `"-"` strings with flags.